src/BlockTemplatesController.php

```diff
@@ -98,25 +206,63 @@ public function get_block_templates() {
 		$template_files = BlockTemplateUtils::gutenberg_get_template_paths( $this->templates_directory );
 		$templates      = array();
 
+		// Check if the template has been saved in the database first.
```

Comment:
I'm having a bad time understanding this comment. This code is fetching all WC templates from the database right? So what about replacing it with something along these lines?
```suggestion
		// First, retrieve the templates which are saved in the database.
```
I might be misunderstanding the code, in that case, feel free to ignore. :slightly_smiling_face:

Reply:
Yes your wording makes a bit more sense :)